### PR TITLE
CE merge

### PR DESCRIPTION
**Pull request by dustinmm80**
This is the final merge for the API client. This merge will be tied to a pre-release.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,13 @@
-FROM ruby:1.9.3
+FROM ruby:2.3
```
Comment (jtuttle) [inline Dockerfile]: Gemfile says Ruby 2.2.6. Should this match?

Comment (jtuttle) [inline Dockerfile]: Going to update Gemfile to 2.3.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+version: '2.1'
+services:
+  postgres:
+    image: postgres:9.3
+
+  conjur:
+    image: registry.tld/possum:0.1.0-stable
```
Comment (jtuttle) [inline docker-compose.yml]: This is the only remaining reference to `possum` in this project. I believe the `conjur` image is in the registry now so should we switch over to that?

Comment (jtuttle) [inline docker-compose.yml]: Changed this to `conjur`.

**Comment by dustinmm80** [comment]
Thanks a lot @jtuttle! I'll get `5.0.0.pre` to RubyGems via Jenkins and we can iterate there briefly before `5.0.0` release.

**Pull request merged**